lib/tools/adb-emu-commands.js

```diff
@@ -43,13 +28,13 @@ emuMethods.fingerprint = async function (fingerprintId, udid = undefined) {
   if (parseInt(level, 10) < 23) {
     log.errorAndThrow(`Device API Level must be >= 23. Current Api level '${level}'`);
   }
-  await this.checkEmulatorConnected(udid);
+  await this.isEmulatorConnected();
```

Comment:
my bad! that's for doing things before having coffee. The idea was to throw an exception, I forgot to add it. Fixing it.